biscuit-capi/tests/capi.rs

```diff
@@ -161,15 +161,50 @@ fn serialize_keys() {
 
             KeyPair * kp = key_pair_new((const uint8_t *) seed, strlen(seed), 0);
             printf("key_pair creation error? %s\n", error_message());
-            PublicKey* pubkey = key_pair_public(kp);
+            PublicKey * pubkey = key_pair_public(kp);
 
             key_pair_serialize(kp, priv_buf);
             public_key_serialize(pubkey, pub_buf);
 
+            const char * pub_pem = public_key_to_pem(pubkey);
+            printf("public key pem: %s\n", pub_pem);
+
+            PublicKey * pubkey2 = public_key_from_pem(pub_pem);
```

Comment:
is there a way to test public keys for equality? that would be a worthwhile addition to the test.

Reply:
I've added this feature here: [`99618b0` (#280)](https://github.com/eclipse-biscuit/biscuit-rust/pull/280/commits/99618b068b907853d00267930292fee539e3448b)